src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -113,8 +96,20 @@ class RequestHandlerComponent extends Component
      */
     public function __construct(ComponentRegistry $registry, array $config = [])
     {
+        if (!isset($config['viewClassMap'])) {
+            $config['viewClassMap'] = [
+                'json' => 'Json',
+                'xml' => 'Xml',
+                'ajax' => 'Ajax'
+            ];
+        }
+        if (!isset($config['inputTypeMap'])) {
+            $config['inputTypeMap'] = [
+                'json' => ['json_decode', true],
+                'xml' => [[$this, 'convertXml']],
+            ];
+        }
```

Comment:
Default config is merged. I didn't want merging as with a merge there is no east way to remove defaults.